server/controllers/submission.js

```diff
@@ -0,0 +1,146 @@
+const Submission = require("../models/Submission");
+const Contest = require("../models/Contest");
+const asyncHandler = require("express-async-handler");
+const uploadMultiImage = require("../utils/uploadFiles");
+
+exports.createSubmission = asyncHandler(async (req, res, next) => {
+    
+    const { creator, contest_id, is_active, date_Created } = req.body;
+   
+    let contest = await Contest.findById(contest_id);
+
+    //Grabbing AWS URL to import into submission model
+    multiImageLocationArray = [];
+    let fileArray = req.files,
+        fileLocation;
+    for (let i = 0; i < fileArray.length; i++) {
+        fileLocation = fileArray[i].location;
+        multiImageLocationArray.push(fileLocation);
+    }
+    let files = multiImageLocationArray
+    //Creating a new submission with aws url array and submission data received from front end
+    if (contest) {
+        const createdSubmission = await Submission.create({
+            creator,
+            contest_id,
+            files,
+            is_active,
+            date_Created,
+        });
+        //Creating a finding and updating contest to apply submission id to array within contest
+        if (createdSubmission) {
+            let submissionArr = contest.submissions;
+            submissionArr.push(createdSubmission._id);
+            contest.submissions = submissionArr;
+
+            const title =contest.title;
+            const price =contest.price;
+            const description = contest.description;
+            const end_date = contest.end_date
```

Comment:
i would try to stick to camelCase for variables

```suggestion
            const endDate = contest.end_date
```

Reply:
I agree, I will fix that right away